templates/.circleci/config.yml

```diff
@@ -220,12 +221,23 @@ workflows:
             #   branches:
             #     only:  # only branches matching the below regex filters will run
             #       - /^master$/
+        
+        - wait_for_approval:
+            type: approval
+            requires:
+              - deploy_staging
 
+        - queue/block_workflow:
```

Comment:
I think this requires a circleci api key to be added as an environment variable, so we would need to do this in the project makefile.

Reply:
I think we already do:

https://github.com/commitdev/zero-deployable-react-frontend/blob/dbe300be4341be431a2bfb6ae380f7a3cf2426f2/Makefile#L3

https://github.com/commitdev/zero-deployable-react-frontend/blob/dbe300be4341be431a2bfb6ae380f7a3cf2426f2/zero-module.yml#L18

Reply:
right nvm! I see what you mean.

`curl -X POST --header "Content-Type: application/json" -d '{"name":"CIRCLECI_API_KEY", "value":"${CIRCLECI_API_KEY}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_API_KEY}`